public/app/plugins/datasource/cloudwatch/datasource.js

```diff
@@ -232,15 +232,23 @@ function (angular, _, moment, dateMath, kbn, templatingVariable, CloudWatchAnnot
       if (ebsVolumeIdsQuery) {
         region = templateSrv.replace(ebsVolumeIdsQuery[1]);
         var instanceId = templateSrv.replace(ebsVolumeIdsQuery[2]);
-        var instanceIds = [
-          instanceId
-        ];
+        var instanceIds = [];
+        if(instanceId.indexOf(",") > -1) {
+          var instanceIdArray = instanceId.replace("{","").replace("}","").split(",");
```

Comment:
this is not the way to do it. pass special formater into templateSrv.replace(ebsVolumeIdsQuery[2]) so that it can return the raw value array